test/functional/feature_block.py

```diff
@@ -82,46 +71,42 @@ def serialize(self, with_witness=False):
         return r
 
     def normal_serialize(self):
-        r = b""
-        r += super(CBrokenBlock, self).serialize()
-        return r
+        return super().serialize()
 
-class FullBlockTest(ComparisonTestFramework):
-    # Can either run this test as 1 node with expected answers, or two and compare them.
-    # Change the "outcome" variable from each TestInstance object to only do the comparison.
+class FullBlockTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
         self.setup_clean_chain = True
+        self.extra_args = [[]]
+
+    def run_test(self):
+        node = self.nodes[0]  # convenience reference to the node
+
+        network_thread_start()
```

Comment:
nit: Could mention in a comment that this is only needed because `connect_p2p` assumes the thread is already running.

nit: Could rename `connect_p2p` to `reconnect_p2p`, since it only works when the thread is already running?

Reply:
Sounds good. Done both.